pcsx2/gui/Panels/CpuPanel.cpp

```diff
@@ -251,8 +251,8 @@ void Panels::CpuPanelEE::ApplyConfigToGui( AppConfig& configToApply, int flags )
 	m_panel_RecEE->Enable(!configToApply.EnablePresets);
 	m_panel_RecIOP->Enable(!configToApply.EnablePresets);
 
-	m_check_EECacheEnable ->SetValue(recOps.EnableEECache);
-	m_check_EECacheEnable->Enable(!configToApply.EnablePresets);
+	m_check_EECacheEnable->SetValue(recOps.EnableEECache);
+	m_check_EECacheEnable->Enable(!configToApply.EnablePresets && !m_panel_RecEE->GetSelection());
```

Comment:
> Maybe worth commenting that EE cache is for the interpreter only?

The tooltip text above has already mentioned it as ``Interpreter only`` and hence I feel it's a bit redundant to be mentioned again.

> Personally I'd prefer m_panel_RecEE->GetSelection() == 0 here, to me !m_panel_RecEE->GetSelection() implies you couldn't get a selection, whereas m_panel_RecEE->GetSelection() == 0 implies item number 0. But maybe I'm being too picky.

I think it's more logical the current way. do notice the variable naming ``m_panel_RecEE``. So I thought it'd be ideal to have it as ``!m_panel_RecEE->GetSelection()`` since it would mean the recompiler is not being used but rather the interpreter. just make a comment if you disagree with the logic and I'll change it back to the one you have suggested.